README.md

```diff
@@ -393,24 +393,70 @@ get removed from the DOM, and if so should there be cases where it remains?
 
 The standard toast will come with these default styles,
 which developers will be able to change to customize look and feel.
+They are similar to [the default styles for `<dialog>`](https://html.spec.whatwg.org/multipage/rendering.html#flow-content-3),
+which are very minimal.
+
+```css
+std-toast {
+    position: fixed;
+    offset-block-end: 0;
+    offset-inline-end: 0;
+    border: solid;
+    padding: 1em;
+    background: white;
+    color: black;
+    z-index: 1;
+}
 
-#### `std-toast:not([open])`
-
-- `display: none`: this will keep the toast hidden from view and searching
-when the `open` attribute is not present.
+std-toast:not([open]) {
+    display: none;
+}
+```
 
-#### `std-toast[open]`
+TODO: we may want additional default styles for the different `type=""` values.
 
-TODO: figure out some default styles, and state them here.
-Additionally figure out default styles for types.
+TODO: the choice of block/inline-end positioning
+(lower-right in left-to-right, horizontal writing modes)
+is not final and will likely be changed.
+It also interacts with the desire for easier positioning discussed in the attributes section.
+Discuss in [#13](https://github.com/jackbsteinberg/std-toast/issues/13) and [#39](https://github.com/jackbsteinberg/std-toast/issues/39).
 
 ### Appearance customization
 
-TODO([#20](https://github.com/jackbsteinberg/std-toast/issues/20)): explain any API for appearance customization,
-beyond just normal CSS.
-For example,
-a CSS shadow part for the close button,
-or some CSS variables.
+Appearance can be customized using normal CSS.
+For example, to change the colors, you could do
+
+```css
+std-toast {
+    border-color: blue;
+    background-color: yellow;
+    color: red;
+}
+```
+
+To completely remove all default styling (useful for library authors),
+use
+
+```css
+std-toast {
+    all: unset;
+}
+```
+
+To style an action button (if present),
+use the `[slot="action"]` selector:
+
+```css
+std-toast [slot="action"] {
+    color: blue;
+    text-transform: uppercase;
+}
+```
+
+(Note: we could offer an alternate syntax of `std-toast::part(action) { ... }`.
+Discuss in [#20](https://github.com/jackbsteinberg/std-toast/issues/20).)
+
+TODO: link to demos.
```

Comment:
The demos will be hosted here (https://jackbsteinberg.github.io/std-toast/) after review, depending on the review timeline we should maybe just add this in

Reply:
Let's add in the link as part of #45.